analyses/molecular-subtyping-HGG/00-HGG-select-pathology-dx.Rmd

```diff
@@ -0,0 +1,185 @@
+---
+title: "Select pathology diagnoses for inclusion"
+output: 
+  html_notebook:
+    toc: TRUE
+    toc_float: TRUE
+author: Jaclyn Taroni for ALSF CCDL
+date: 2020
+---
+
+## Background
+
+Originally, we subtyped tumors in this module if the specimen satisfied one of the following criteria:
+
+1. A defining lesion was identified in the SNV consensus file (H3 K28M or G35R/V)
+2. The `short_histology` was `HGAT`.
+
+In an upcoming release, `integrated_diagnosis`, which can be updated as the result of subtyping, will be used to populate the `short_histology` column (see [#748](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/748)).
+Thus, molecular subtyping modules need to be upstream of `short_histology` and use the `pathology_diagnosis` and `pathology_free_text_diagnosis` fields.
+This change for this module is tracked in [#754](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/754).
+
+Filtering on the basis of `short_histology == HGAT` is more straightforward than using the pathology diagnosis fields, so we include this notebook to put together the terms in `pathology_diagnosis` and `pathology_free_text_diagnosis`.
+
+We will use the 2016 WHO Classification as our guide ([Louis et al. _Acta Neuropathol._ doi: 10.1007/s00401-016-1545-1](10.1007/s00401-016-1545-1)) and take a look at the current version of the histology file (`release-v17-20200908`).
+
+## Set up
+
+```{r}
+library(tidyverse)
+```
+
+### Directories and files
+
+We're going to tie this to a specific release.
```

Comment:
I don't think we will be in a situation where we want to look at the contents of the histologies files on the basis of `short_histology == "HGAT"` again, once `short_histology` fully depends on `molecular_subtype` (planned for next release). I can definitely see a situation where we would update the JSON (that's the rationale for including it rather than hardcoding in `02`!) but I don't think the process for getting those terms would be the same.

>  If the vocabulary were to change, would we update that manually and deprecate this notebook?

All that to say - I think you're correct here.